README.md

```diff
@@ -112,6 +112,19 @@ $ make nodelink-controller
      new instance is created. Followed by new node joining the cluster
      and turning in `Ready` state.
 
+## Creating machines
+
+You can create a new machine by [applying a manifest representing an instance of the machine CRD](docs/examples/machine.yaml)
+
+The `machine.openshift.io/cluster-api-cluster` label will be used by the controllers to lookup for the right cloud instance.
+
+You can set other labels to provide a convenient way for users and consumers to retrieve groups of machines:
+```
+machine.openshift.io/cluster-api-machine-role: worker
+machine.openshift.io/cluster-api-machine-type: worker
```

Comment:
qq @enxebre : could users set custom labels like `foo: bar` or it always needs to follow the above convention `machine.openshift.io/cluster-api-machine-role` such that consumers can know how to filter?

Reply:
@DanyC97 these are arbitrary labels in the sense that the core controller does not use them today. You might still want to use the `machine.openshift.io/` namespace for consistency and searchability. However you could change them or/and add new labels at your convenience

Reply:
thank you @enxebre for taking the time to respond 👍